letsencrypt/rootfs/etc/services.d/lets-encrypt/run

```diff
@@ -91,6 +91,11 @@ elif [ "${DNS_PROVIDER}" == "dns-gandi" ]; then
     fi
     PROVIDER_ARGUMENTS+=("--authenticator" "certbot-plugin-gandi:dns" "--certbot-plugin-gandi:dns-credentials" /data/dnsapikey)
 
+# Hetzner
+elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-hetzner" ]; then
+    bashio::config.require 'dns.hetzner_api_token'
```

Comment:
```suggestion
```
Shoot sorry I missed this the first time I guess but don't do config validation in here. Config validation like has to be in `cont-init.d` if it's necessary. The problem with doing it in here is that will cause the run script to fail and when it fails then s6 just keeps restarting it.

Reply:
I put it there because other plugins already do it in there (directadmin, njalla, azure to name a few). I don't see anything similar in `cont-init.d`. If you're OK with removing it entirely I can do that.

Reply:
Hm yea I see others are doing it. Ok fair enough we can leave it. It may put the addon in a restart loop but I suppose at least the user will clearly see what they did wrong.